README.md

```diff
@@ -21,15 +21,37 @@ module.exports = {
                 visible: 20000    
             },
             capabilities: {
-                browserName: 'chrome'
+                browserName: 'chromium'
             }
         },
         pageObject: new App()
     }
 }
 ```
 
-## screenshot strategy
+## Connect to playwright server
+In order to connect to playwright server pass _wsEndpoint_ property in capabilities object
+```typescript
+{
+    capabilities: {
+        browserName: 'chromium',
+        wsEndpoint: 'ws://127.0.0.1:60291/2bd48ce272de2b543e4c8c533f664b83'    
```

Comment:
as I understand it right - to connect to browser using websocet connection we need to hardcode that value here, am I right?

Reply:
yep, I understand, I just curious how we would know websocet id before browser session start if we going to hardcode it in the config file?

Reply:
in case of playwright server, it returns this url vis wsEndpoint method.
as for cdp endpoint, we can start chrome-like browser with desired debugging port

Reply:
got it